libnavigation-base/build.gradle

```diff
@@ -19,6 +19,10 @@ dependencies {
     api dependenciesList.mapboxSdkGeoJSON
 
     implementation dependenciesList.supportAnnotation
+    api dependenciesList.coroutinesCore
+    api dependenciesList.coroutinesAndroid
```

Comment:
Not sure if `coroutinesAndroid` should be exposed from `libnavigation-base`. For now, I think we could include it as part of `libtrip-service` and revisit later. What do you think?

In any case, as commented above, noticed that `TripServiceImpl` and `NavigationNotificationService` are living under `libandroid-navigation` module so feel free to move the dependency there if it helps testing, we can shuffle dependencies and move classes around later.

Reply:
Open for discussion. If it's part of base, then other modules can use it, otherwise you'll have to add that dependency to all modules that want to use it.

Reply:
> otherwise you'll have to add that dependency to all modules that want to use it.

Imo, that's okay. Coroutines are an implementation detail and shouldn't be part of the contract.